src/mainAppBar/mainAppBar.component.tsx

```diff
@@ -84,7 +84,9 @@ const styles = (theme: Theme): StyleRules =>
       padding: 4,
       margin: theme.spacing(1),
       '& img': {
-        height: 24,
+        maxHeight: `calc(${
+          (theme as UKRITheme).mainAppBarHeight
+        } - 2 * 4px - 2 * ${theme.spacing(1)}px)`,
```

Comment:
How did you calculate this?

Reply:
That shows I should add a comment! It's just minusing the margin and padding from the overall height, so the logo can take up all the remaining space. everything is times 2 because things have bottom and top padding & margin

Reply:
Ah I'm with you now